manage_people/migrations/0001_mp_initial.py

```diff
@@ -20,71 +21,88 @@
+    ('manage_people', '*', 'StudentEnrollment', False),
+    ('manage_people', '*', 'TaEnrollment', True),
+    ('manage_people', '*', 'TeacherEnrollment', True),
+    ('manage_people', '*', 'Teaching Staff', True),
+    ('manage_people', 'gse', 'Help Desk', False),
+    ('manage_people', 'hks', 'Course Head', False),
+    ('manage_people', 'hks', 'Course Support Staff', False),
+    ('manage_people', 'hks', 'DesignerEnrollment', False),
+    ('manage_people', 'hks', 'Librarian', False),
+    ('manage_people', 'hks', 'TaEnrollment', False),
+    ('manage_people', 'hks', 'Teaching Staff', False),
+    ('manage_people', 'hls', 'Account Observer', False),
+    ('manage_people', 'hls', 'Course Head',  False),
+    ('manage_people', 'hls', 'Course Support Staff', False),
+    ('manage_people', 'hls', 'Department Admin', False),
+    ('manage_people', 'hls', 'DesignerEnrollment', False),
+    ('manage_people', 'hls', 'Faculty', False),
+    ('manage_people', 'hls', 'Help Desk', False),
+    ('manage_people', 'hls', 'Librarian', False),
+    ('manage_people', 'hls', 'TaEnrollment', False),
+    ('manage_people', 'hls', 'TeacherEnrollment', False),
+    ('manage_people', 'hls', 'Teaching Staff', False)
 ]
 
 
-
 def populate_manage_people_role(apps, schema_editor):
     ManagePeopleRole = apps.get_model('manage_people', 'ManagePeopleRole')
     fields = ('user_role_id', 'canvas_role_label', 'xid_allowed')
     with transaction.atomic():  # wrap all the inserts in a transaction
         for values in MANAGE_PEOPLE_ROLE_DATA:
             ManagePeopleRole.objects.create(**dict(zip(fields, values)))
 
+
 def create_lti_permissions(apps, schema_editor):
-    LtiPermission = apps.get_model('lti_permissions', 'LtiPermission')
+    try:
+        LtiPermission = apps.get_model('lti_permissions', 'LtiPermission')
+    except LookupError:
+        # If the LtiPermission table doesn't exist, return from here
+        return
+
     fields = ('permission', 'school_id', 'canvas_role', 'allow')
 
     for permission in LTI_PERMISSIONS_DATA:
         LtiPermission.objects.create(**dict(zip(fields, permission)))
 
+
 def reverse_permissions_load(apps, schema_editor):
-    LtiPermission = apps.get_model('lti_permissions', 'LtiPermission')
+    try:
+        LtiPermission = apps.get_model('lti_permissions', 'LtiPermission')
+    except LookupError:
+        # If the LtiPermission table doesn't exist, return from here
+        return
+
     LtiPermission.objects.filter(permission='manage_people').delete()
 
+
 def reverse_manage_people_role_load(apps, schema_editor):
     ManagePeopleRole = apps.get_model('manage_people', 'ManagePeopleRole')
     ManagePeopleRole.objects.all().delete()
 
+
 class Migration(migrations.Migration):
 
-    dependencies = [
-         ('lti_permissions', '0001_initial'),
-    ]
+    dependencies = []
+
+    # tlt-2650: we need to check whether lti_permissions is part of
+    # INSTALLED_APPS and skip this migration if the LtiPermission model is
```

Comment:
Here's another place to clarify things, or else remove the first element in the `operations` list below.